model.py

```diff
@@ -598,6 +598,7 @@ class DataSource(Base):
     ADOBE = "Adobe DRM"
     PLYMPTON = "Plympton"
     OA_CONTENT_SERVER = "Library Simplified Open Access Content Server"
+    GOOGLE = "Google Oauth"
```

Comment:
It seems weird for this to be in data sources. It's more like Millenium Patron than the book data sources, but I don't think there's anything in core for that.